proto/connectrpc/conformance/v1alpha1/service.proto

```diff
@@ -131,19 +194,26 @@ message ConformancePayload {
     // The server echos back the request headers it observed here.
     repeated Header request_headers = 1;
     // The timeout observed that was included in the request.
-    optional uint32 timeout_millis = 2;
+    optional uint32 timeout_ms = 2;
+    // The server should echo back all requests received.
+    // For unary and server-streaming requests, this should always contain a single request
+    // For client-streaming and half-duplex bidi-streaming, this should contain
+    // all client requests in the order received and be present in each response.
+    // For full-duplex bidirectional-streaming, this should contain all requests *so far* in the
+    // order received, meaning that this should grow as responses are sent.
```

Comment:
> this should grow as responses are sent.

I would prefer this only be the requests observed since the last response. That way the test case data, in describing the expected responses, doesn't end up huge because earlier requests are repeated in every response.

Reply:
Cool. Updated the comment to reflect that.